client/src/DebugConstants.ts

```diff
@@ -42,9 +42,8 @@ export class DebugConstants {
     public static readonly DEFAULT_SCRIPT_FILE = `defaultScriptFile`;
     public static readonly AUTHENTICATION_SCRIPT_TRUE = `AuthenticationScript enabled="true"`;
     public static readonly AUTHENTICATION_SCRIPT_FALSE = `AuthenticationScript enabled="false"`;
-    public static readonly IAM_URL = "IAM.URL";
+    public static readonly IAM_BASE_URL = "IAM.BaseURL";
```

Comment:
Type will be inferred from usage, but it's always best to declare the type.

Reply:
https://github.com/wso2-extensions/identity-tools-plugin-vscode/pull/7